apps/web/src/components/safe-messages/MsgSigners/MsgSigners.test.tsx

```diff
@@ -31,8 +31,8 @@ describe('MsgSigners', () => {
       proposedBy: {
         value: '',
       },
-      status: SafeMessageStatus.NEEDS_CONFIRMATION,
-      type: SafeMessageListItemType.MESSAGE,
+      status: 'NEEDS_CONFIRMATION' as const,
+      type: 'MESSAGE' as const,
```

Comment:
Does it throw errors for you without the `as const`? For me its working without.
<img width="497" alt="Screenshot 2025-03-31 at 11 19 20" src="https://github.com/user-attachments/assets/4ace9eba-a9f4-4889-982b-8a405035b845" />

Reply:
Initially it was, that's why the codemod is setting it to "as const", not sure why it works without it now.

Reply:
Ok, removed.